cloudfoundry/config-server/create.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source ../common.sh
+
+cf create-service -c '{"git": { "uri": "https://github.com/ilayaperumalg/config-repo"}}' $CONFIG_SERVER_SERVICE_NAME $CONFIG_SERVER_PLAN_NAME cloud-config-server
```

Comment:
yes, that's the plan. Until we merge this in we need a way to refer to an external repo ;)